src/macrofier/engine.mjs

```diff
@@ -844,7 +844,7 @@ function generateSchemas(json, templates, options) {
     else {
       content = content.replace(/.*\$\{schema.seeAlso\}/, '')
     }
-//    content = content.trim().length ? content.trimEnd() : content.trim()
+    content = content.trim().length ? content : content.trim()
```

Comment:
Now I am trimming only if the it end up with whitespace alone. content.trim().length given zero if it is whitespace alone. So I think it will not make issue for any language